packages/UsageAnalysis/files/Test Track/General/Molecule in exported CSV.md

```diff
@@ -0,0 +1,17 @@
+### Molecule column in exported CSV
+1. Open the Dataset: Open the SPGI dataset.
```

Comment:
Link the SPGI dataset to the test case to ease the process

Reply:
added